query/filter/selector.go

```diff
@@ -46,6 +46,22 @@ func (s *Selector) Matches(doc []byte) bool {
 	return false
 }
 
+func (s *Selector) ToSearchFilter() string {
+	switch s.Matcher.Type() {
+	case EQ:
+		return fmt.Sprintf("%s:=%v", s.Field, s.Matcher.GetValue().AsInterface())
```

Comment:
`op = ":="` in every case and
return `fmt.Sprintf("%s%s%v", s.Field, op, s.Matcher.GetValue().AsInterface())`
at the end.
also return "" in the default case